docs/glossary.md

```diff
@@ -14,7 +14,7 @@ The following is a glossary of domain specific terminology. Although benchmarks
 ## Benchmarks
 
 * **stress test benchmark**: a benchmark that is specifically designed to stress a certain part of the compiler. For example, [projection-caching](https://github.com/rust-lang/rustc-perf/tree/master/collector/benchmarks/projection-caching) stresses the compiler's projection caching mechanisms.
-* **real world benchmark**: a benchmark based on a real world crate. These are typically copied as-is from crates.io. For example, [serde](https://github.com/rust-lang/rustc-perf/tree/master/collector/benchmarks/serde) is a popular crate and the benchmark has not been altered from a release of serde on crates.io. 
+* **real world benchmark**: a benchmark based on a real world crate. These are typically copied as-is from crates.io. For example, [serde](https://github.com/rust-lang/rustc-perf/tree/master/collector/benchmarks/serde-1.0.136) is a popular crate and the benchmark has not been altered from a release of serde on crates.io. 
```

Comment:
Change `[serde]` to `[serde-1.0.136]`?

Reply:
I didn't change that because it's more confusing when written that way. This refers to the crate, not the benchmark, and the name of the crate is serde, not serde-1.0.136. 

But if you feel strongly we should change it, I can do that.

Reply:
Ok, it's fine as is.